### DISCUSSION TITLE
a deselect method for Spool

### DISCUSSION THREAD

**Discussion by ahmadtourei** [DASDAE/dascore#444] [Ideas]
For ambient noise interferometry workflow, we often need to exclude parts of the data that include high-amplitude signals or events. Having a `Spool.deselect(time=(t1, t2), distance=(d1, d2))` similar to `Spool.select` method would be beneficial.

**Comment by d-chambers** [comment]
Sounds good to me. It might get a little bit tricky to implement though before we refactor the indexer (which will basically address all the open issues as well).
